packages/axios-extension/README.adoc

```diff
@@ -65,7 +65,11 @@ const provider = createForAbap({
     params: { 'sap-client': client }
 });
 const transportRequestSerivce = provider.getAdtService<TransportRequestService>(TransportRequestService);
-const newTrNumber = await transportRequestSerivce.createTransportRequest('A new transport request number for deployment');
+const newTrNumber = await transportRequestSerivce.createTransportRequest({
+    packageName: 'Z_PACKAGE',
```

Comment:
What does the backend service actually do with the package and BSP name? Does it validate that it exists? If yes, what do I do when I want to create a transport for a new application that does not yet exist?

Reply:
README is to provide example of using ADT service. Instead, I updated the jsdoc for ui5AppName parameter in transportrequest-service.ts

Reply:
I'd prefer getting a bit more information in the `README` as well but not just for this one, so for this PR, I am ok, and will approve, however, please consider a follow up given a little more guidance (like you explanation above) so that consumers have all information where it belongs.